csharp-intermediate-topics/ConstVsReadonlyInCSharp/ConstVsReadonlyInCSharp/MathCalculator.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ConstVsReadonlyInCSharp
+{
+    public class MathCalculator
+    {
+        public double Add(double a, double b)
+        {
+            return a + b;
```

Comment:
Since all of these methods are one line and very simple, the code would be a lot shorter if it used expression body statements.